site/en/tutorials/text/word_embeddings.ipynb

```diff
@@ -597,7 +625,9 @@
         "else:\n",
         "  files.download('vecs.tsv')\n",
```

Comment:
@MarkDaoust 
I'm getting an error here running in public colab: `MessageError: TypeError: Failed to fetch`

Reply:
That's a Colab problem.

Sometimes it works the second time.